ev3dev/core.py

```diff
@@ -61,6 +61,9 @@
 INPUT_AUTO = ''
 OUTPUT_AUTO = ''
 
+# Keep the __version__ in sync with the package version in debian/changelog
+__version___ = '0.7.0'
```

Comment:
Is the triple underscore intentional?

Surely there is a way to do this automatically, e.g. with setuptools?